notification-service/src/app.ts

```diff
@@ -0,0 +1,37 @@
+import 'reflect-metadata';
+import DatabaseConfig from './config/database';
+import * as Express from 'express';
+import * as bodyParser from 'body-parser';
+import NotificationRouter from './routes/notification.route'
+
+class Notification {

```

Comment:
Change this name for something like NotificationApp, just to follow the pattern of previous services

Reply:
Done